.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Sorry I'm a little confused about this too, I think I'm just unfamiliar with how issue templates work. Does this get included in the issue description, and it's up to them to edit it?